crates/stdsimd-test/src/disassembly.rs

```diff
@@ -108,14 +116,18 @@ fn parse_objdump(output: &str) -> HashMap<String, Vec<Function>> {
             instructions.push(Instruction { parts });
         }
 
-        ret.entry(normalize(symbol))
+        ret.entry(symbol)
             .or_insert_with(Vec::new)
             .push(Function {
                 addr: None,
                 instrs: instructions,
             });
     }
 
+    for (_k, v) in &mut ret {
+        assert_eq!(v.len(), 1);
+    }
```

Comment:
So right now these are still `HashMap<String, Vec<Function>>`, but this should become `HashSet<String, Function>` in the future. 

If we are finding multiple duplicated symbols of the same shim, things already went wrong somewhere.